yb-voyager/cmd/importDataToSource.go

```diff
@@ -82,3 +85,40 @@ func initTargetConfFromSourceConf() error {
 	tconf.Uri = sconf.Uri
 	return nil
 }
+
+func packAndSendImportDataToSourcePayload(status string) {
+
+	if !callhome.SendDiagnostics {
+		return
+	}
+	payload := createCallhomePayload()
+	payload.MigrationType = LIVE_MIGRATION_WITH_FALLBACK
```

Comment:
What are the othere possible values? I'm wondering if we should limit this to just OFFLINE/LIVE, and have another field to indicate the fallback/fall-forward behavior

Reply:
Agreed, but what I was worried about is that for the same migration: 
- export-data, import-data will have migration_type = LIVE_MIGRATION
- export-data-from-target, import-data-to-source-replica, import-data-to-source will have migration_type = LIVE_MIGRATION_WITH_FALLBACK/FALLFORWARD. 
Correct me if this is not the case. If it is, it seems a bit weird IMO. But I'm not able to think of a better structure either. The best I can come up with is: just let the migration_type be LIVE_MIGRATION; the fact that it is fall-back/fall-forward can be derived based on the commands run

Reply:
Yes right, it is like this right now.
Hmm, but I think having a "live migration with fall-back/fall-forward" upfront in migration_type is because if someone just did a simple query to check migration_type of the migration_uuid, it should return this as well which will give an idea to the user that fall-forward/fall-back is also used.

Reply:
Makes sense. I can't think of a better approach either. Let's discuss this once in the standup?

Reply:
Made this change for adding a separate field and keeping live-mgiration only in the migration_type